client/src/main/js/controller/dialog/ImagemController.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    angular.module('localModulo').controller('ImagemController', ['imagem', 'editavel', '$rootScope', '$mdDialog', function (imagem, editavel, $rootScope, $mdDialog) {
+        var self = this;
+        this.imagem = imagem;
+        this.editavel = editavel;
+
+        this.excluirImagem = function () {
```

Comment:
se for apenas direcionar essa função para o callback recebido, não precisa de jsdoc. Se for implementar a solução de remover a imagem da lista/local, adicionar jsdoc